apps/front-end/front-office/src/app/reducers/game/game.effects.ts

```diff
@@ -55,12 +60,30 @@ export class GameEffects {
               );
             }
             return this.patientService.getPatientAccommodation(patient.id).pipe(
-              map((accommodation) => {
-                this.redirectToCorrectQuestion(quiz.data.questions[0]);
-                return GameActions.loadGameSuccess({
-                  quiz: quiz.data,
-                  accommodation: accommodation.data,
-                });
+              mergeMap((accommodation) => {
+                return this.quizSessionService
+                  .createQuizSession({
+                    quiz: {
+                      connect: {
+                        id: quiz.data.id,
+                      },
+                    },
+                    patient: {
+                      connect: {
+                        id: patient.id,
+                      },
+                    },
+                  })
+                  .pipe(
```

Comment:
No need to `pipe(x).pipe(y)`, you can `pipe(x, y)`

Reply:
there's a mergeMap